cmd/appgw-ingress/env_variables.go

```diff
@@ -1,40 +0,0 @@
-// -------------------------------------------------------------------------------------------
```

Comment:
@akshaysngupta as you are working on this - would you be so kind to `git mv A B` so the history of these files is preserved?

Reply:
Typically if you use the operating system's primitives to rename a file - git sees it as delete + add;  So you have to use `git mv` instead (which I think is the same on Windows as it is on Linux)